### ISSUE TITLE
Inverted icon colors on 2023.2

### ISSUE THREAD

**Issue by jacekgajek** [MCMicS/jenkins-control-plugin#503]
![image](https://github.com/MCMicS/jenkins-control-plugin/assets/68021034/a6db80f1-0156-402f-ab0e-088aeaf76a39)

```
IntelliJ IDEA 2023.2 (Ultimate Edition)
Build #IU-232.8660.185, built on July 26, 2023
Licensed to Jacek Gajek
You have a perpetual fallback license for this version.
Subscription is active until July 29, 2024.
Runtime version: 17.0.7+7-b1000.6 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Linux 5.19.0-50-generic
Jenkins Plugin 0.13.18-2023.2
GC: G1 Young Generation, G1 Old Generation
Memory: 8000M
Cores: 24
Registry:
    debugger.new.tool.window.layout=true
    ide.experimental.ui=true
    scala.erase.compiler.process.jdk.once=false

Non-Bundled Plugins:
    com.github.leomillon.uuidgenerator (4.5.1)
    de.netnexus.camelcaseplugin (3.0.12)
    IdeaVIM (2.4.1)
    org.turbanov.run.configuration.as.action (1.5.1)
    commit-prefix-plugin (1.2.1)
    com.github.stevejagodzinski.akka-log-inspection-plugin (1.2.9)
    Jenkins Control Plugin (0.13.18-2023.2)
    String Manipulation (9.10.0)
    woaini.liquibase.plugin (1.3-SNAPSHOT)
    PlantUML integration (6.3.0-IJ2023.2)
    dev.nx.console (1.10.0)
    com.github.iguissouma.nxconsole (0.58.0)
    Pythonid (232.8660.185)
    com.rspn.git (1.9)
    kotest-plugin-intellij (1.3.69-IC-232-EAP-SNAPSHOT)
    com.intellij.ml.llm (232.9353)
    org.intellij.scala (2023.2.17)
    zio.intellij.zio-intellij (2023.2.27.0)
    aws.toolkit (1.74-232)
    org.jetbrains.plugins.docker.gateway (232.8660.202)

Kotlin: 232-1.9.0-IJ8660.185
Current Desktop: KDE
```

**Comment by MCMicS** [comment]
Thanks. I'll fix it

**Comment by MCMicS** [comment]
Please test following EAP: I now use the prefereed Idea colors. If you have any issues please ask again

https://plugins.jetbrains.com/plugin/6110-jenkins-control/versions/eap/370934

**Comment by MCMicS** [comment]
released with [0.13.19](https://plugins.jetbrains.com/plugin/6110-jenkins-control/versions/stable/408555)
